docs/manual/common/welcome/WhereToStart.md

```diff
@@ -0,0 +1,14 @@
+# Where to begin
+
+It takes time to adopt a new way of thinking about and solving problems. Lagom provides guidance with its framework, but ultimately it is up to you &#8212; and possibly others in your organization &#8212; to apply the principles in your domain. You will be able to do this successfully by fully understanding what motivates use of microservices and how they impact not just the resulting system, but the development process itself.
```

Comment:
GTK. I tried the double dash and wasn't getting the m-dash, so in this topic, I fell back to HTML.